src/vocabularyChecker/detectWrongSyntax.ts

```diff
@@ -20,16 +20,30 @@ export function collectInvalidWordsInIssues(text: string) {
 	const positions: Position[] = [];
 	for (const word of invalidWords) {
 		if (!(allowedWords.includes(word.toLowerCase()))) {
-			const temp: Position = {
-				fromIndex: text.indexOf(word),
-				toIndex: (text.indexOf(word) + word.length),
-			};
-			positions.push(temp);
-			addIssue("INVALID_WORD", temp, {word});
+			const allPositionsOfInvalidWord = getPositionsOfInvalidWord(word, text);
+			for (const pos of allPositionsOfInvalidWord) {
+				positions.push(pos);
```

Comment:
As far as I can tell the variable position and therefore this push doesn't do anything (anymore)?

Reply:
I will have to test this, but I think you are right, if I am not mistaken.